content/general/concepts/data-mining/data-mining.md

```diff
@@ -0,0 +1,30 @@
+---
+Title: 'Data Mining'
+Description: 'Data mining is the process of applying algorithms to search for patterns within collections of data.'
+Subjects:
+  - 'Data Science'
+  - 'Data Visualization'
+  - 'Machine Learning'
+Tags:
+  - 'Data Science'
+  - 'Deep Learning'
+CatalogContent:
+  - 'learn-python-3'
+  - 'paths/data-science'
+---
+
+**Data mining** is the process of applying algorithms to search for patterns within collections of data.
+
+Fundamentally, data mining is the deployment of an automated process for analyzing large amounts of data that otherwise could not be addressed manually. This process generally involves several steps that may include data collection, data cleaning and validation (to address any errors, inconsistencies, and other quality issues), model development and testing, and final deployment.
```

Comment:
```suggestion
Fundamentally, data mining is the deployment of an automated process for analyzing large amounts of data that otherwise could not be analyzed manually. This process generally involves several steps that may include data collection, data cleaning and validation (to address any errors, inconsistencies, and other quality issues), model development and testing, and final deployment.
```